rules/game_process.tex

```diff
@@ -90,10 +89,9 @@ \subsection{Robot States}
 Once the game controller operator has confirmed the \texttt{standby} state, the referee should wait a random time between 10 to 120 seconds before announcing the transition to \texttt{ready}. 
 Teams should expect that the delay before the referee signals the transition from \texttt{standby} to \texttt{ready} will be different every time.
 To announce the transition from \texttt{standby} to \texttt{ready} state, the referee will raise both hands over their head.
-Red gloves will be worn by the referee, though teams are encouraged not to rely on this visual cue as the gloves could be removed in a following year.
```

Comment:
I think the red gloves for CS are still in discussion? So this sentence has to be updated then.